streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -5333,6 +5333,12 @@ object ZStream extends ZStreamPlatformSpecificConstructors {
   def logInfo(message: => String)(implicit trace: ZTraceElement): ZStream[Any, Nothing, Unit] =
     ZStream.fromZIO(ZIO.logInfo(message))
 
+  /**
+   * Logs the specified message at the informational log level.
```

Comment:
here its "informational", in ZManaged its "runtime"

Reply:
Fixed in my latest commit.